src/views/notifications/Latest.vue

```diff
@@ -0,0 +1,324 @@
+                                {{ this.validation_error?.color[0] }}
+                              </div>
+                            </div>
+                          </div>
+                          <div class="col-span-12 xxl:col-span-6 mb-4">
+                            <div>
+                              <label class="form-label">
+                                Type
+                              </label>
+                              <TailSelect
+                                v-model="this.single_notification.type"
+                                :class="'form-control' + (this.validation_error?.type != null ? ' border-theme-6' : '')"
+                                :options="{
+                                search: true,
+                                classNames: 'w-full'
+                              }"
+                              >
+                                <option value=1>Icon</option>
+                                <option value=2>User</option>
+                              </TailSelect>
+                              <div v-if="this.validation_error?.type != null" class="text-theme-6 mt-2 mb-4">
+                                {{ this.validation_error?.type[0] }}
+                              </div>
+                              {{ this.single_notification.type }}
+                            </div>
+                          </div>
+                          <div class="col-span-12 xxl:col-span-12">
+                            <div class="flex items-center">
+                              <div class="border-l-2 border-theme-1 pl-4">
+                                <a href="" class="font-medium">
+                                  Has the notification already been seen?
+                                </a>
+                                <div class="text-gray-600">
+                                  Select this option to mark the notification as read.
+                                </div>
+                              </div>
+                              <input class="form-check-switch ml-auto" type="checkbox" v-model="this.single_notification.seen">
+                            </div>
+                          </div>
+                          <div class="col-span-12 xxl:col-span-12">
+                            <hr class="my-5">
+                          </div>
+                          <div class="col-span-12 xxl:col-span-6 mb-4">
+                            <div>
+                              <label for="update-icon-notification" class="form-label">
+                                Icon
+                              </label>
+                              <input
+                                id="update-icon-notification"
+                                type="text"
+                                :class="'form-control' + (this.validation_error?.icon != null ? ' border-theme-6' : '')"
```

Comment:
Unnecessary `!= null`

Reply:
Solved in https://github.com/KrbDevelopment/Articly-Frontend/commit/5172bc53bad0121ee21691def6813959202839fa